dropshipping/demo/ir_attachment_post.xml

```diff
@@ -0,0 +1,89 @@
+  </record>
+  <record id="ir_attachment_1304" model="ir.attachment">
+    <field name="name">DALL·E 2025-03-18 15.39.05 - A high-resolution soft and fuzzy texture background in a light green pastel color theme. The texture should appear smooth and airy, with a delicate, b.jpg</field>
+    <field name="datas" type="base64" file="dropshipping/static/src/binary/ir_attachment/1303-DALL·E2025-03-1815.39.05-Ahigh-resolutionsoftandfuzzytexturebackgroundinalightgreenpastelcolortheme.Thetextureshouldappearsmoothandairy,withadelicate,b.webp"/>
+    <field name="res_id" ref="ir_attachment_1303"/>
+    <field name="public" eval="True"/>
+  </record>
+  <record id="ir_attachment_1371" model="ir.attachment">
+    <field name="name">DALL·E 2025-03-18 16.58.51 - A high-resolution, seamless orange background with a smooth and subtle wavy pattern. The entire image should be filled with gentle, flowing curves tha.webp</field>
+    <field name="datas" type="base64" file="dropshipping/static/src/binary/ir_attachment/1371-DALL·E2025-03-1816.58.51-Ahigh-resolution,seamlessorangebackgroundwithasmoothandsubtlewavypattern.Theentireimageshouldbefilledwithgentle,flowingcurvestha.webp"/>
+    <field name="public" eval="True"/>
+  </record>
+  <record id="ir_attachment_1372" model="ir.attachment">
+    <field name="name">DALL·E 2025-03-18 16.58.51 - A high-resolution, seamless orange background with a smooth and subtle wavy pattern. The entire image should be filled with gentle, flowing curves tha.jpg</field>
+    <field name="datas" type="base64" file="dropshipping/static/src/binary/ir_attachment/1371-DALL·E2025-03-1816.58.51-Ahigh-resolution,seamlessorangebackgroundwithasmoothandsubtlewavypattern.Theentireimageshouldbefilledwithgentle,flowingcurvestha.webp"/>
+    <field name="res_id" ref="ir_attachment_1371"/>
+    <field name="public" eval="True"/>
+  </record>
+  <record id="ir_attachment_1373" model="ir.attachment">
+    <field name="name">DALL·E 2025-03-18 17.00.41 - A high-resolution, seamless pink background with an ultra-smooth and uniform texture. The entire image should be filled with a consistent, soft pink c.webp</field>
+    <field name="datas" type="base64" file="dropshipping/static/src/binary/ir_attachment/1373-DALL·E2025-03-1817.00.41-Ahigh-resolution,seamlesspinkbackgroundwithanultra-smoothanduniformtexture.Theentireimageshouldbefilledwithaconsistent,softpinkc.webp"/>
+    <field name="public" eval="True"/>
+  </record>
+  <record id="ir_attachment_1374" model="ir.attachment">
+    <field name="name">DALL·E 2025-03-18 17.00.41 - A high-resolution, seamless pink background with an ultra-smooth and uniform texture. The entire image should be filled with a consistent, soft pink c.jpg</field>
+    <field name="datas" type="base64" file="dropshipping/static/src/binary/ir_attachment/1373-DALL·E2025-03-1817.00.41-Ahigh-resolution,seamlesspinkbackgroundwithanultra-smoothanduniformtexture.Theentireimageshouldbefilledwithaconsistent,softpinkc.webp"/>
```

Comment:
I guess you can only define one attachment out of 2 then, no? If you use wepb?
Like ir_attachment_1374 and ir_attachment_1373 are now the same and could be merged